android/sdl_android/src/androidTest/java/com/smartdevicelink/test/Validator.java

```diff
@@ -3929,4 +3930,56 @@ public static boolean validateStabilityControlStatus(VehicleDataResult status1,
         }
         return status1.getDataType().equals(status2.getDataType()) && status1.getResultCode().equals(status2.getResultCode());
     }
+
+    public static boolean validateConfigurableKeyboards(ConfigurableKeyboards keyboard1, ConfigurableKeyboards keyboard2) {
+        if (keyboard1 == null) {
+            return (keyboard2 == null);
+        }
+        if (keyboard2 == null) {
+            return (keyboard1 == null);
+        }
+        return keyboard1.getKeyboardLayout().equals(keyboard2.getKeyboardLayout()) && keyboard1.getNumConfigurableKeys().equals(keyboard2.getNumConfigurableKeys());
+    }
+
+    public static boolean validateKeyboardCapability(KeyboardCapabilities keyboard1, KeyboardCapabilities keyboard2) {
```

Comment:
@kostyaBoss I would rename the function on line 3944 to `validateKeyboardCapabilities`.